src/napari_imagej/types/converters/images.py

```diff
@@ -47,7 +47,7 @@ def _dataset_to_image(image: Any) -> Image:
         data=ij().py.from_java(imgplus.getImg()),
         name=imgplus.getName(),
     )
-    if imgplus.getColorTableCount():
+    if imgplus.getColorTableCount() and imgplus.getColorTable(0):
```

Comment:
Starting in Python 3.8, you can use the `:=` operator:
```python
    if imgplus.getColorTableCount() and color_table := imgplus.getColorTable(0):
        kwargs["colormap"] = _color_table_to_colormap(color_table)
```
Side note: from the CI config, it looks like we are currently supporting/testing [3.7, 3.10]. But the environment.yml, dev-environment.yml, and pyproject.toml do not declare a minimum Python version. They should do that, no?

Reply:
If I'm reading the source code of napari correctly, the napari project is requiring 3.8+ now. It seems like 3.8 is pretty ubiquitously available now. Even my Synology NAS box is running Python 3.8.6 at this point. :laughing: Shall we bump the minimum required version so that you can use this helpful construct?

Reply:
Oh, about the `imgplus.getColorTableCount()` truthiness: I think it would be clearer to put `> 0`. After all, it could theoretically return -1 like some C/Java code does. (OTOH, it could theoretically also return None in which case the `> 0` would raise an exception... but since the Java type signature returns `int` it can never actually return None in practice barring some kind of JPype bug :shrug:).

Reply:
> Side note: from the CI config, it looks like we are currently supporting/testing [3.7, 3.10]. But the environment.yml, dev-environment.yml, and pyproject.toml do not declare a minimum Python version. They should do that, no?

Yup, we're supporting 3.7. Unfortunately, napari-imagej hasn't migrated to a pure `pyproject.toml` usage like we did for the other projects. I filed #146 to fix this.

> Even my Synology NAS box is running Python 3.8.6 at this point. laughing Shall we bump the minimum required version so that you can use this helpful construct?

Oh, yeah, let's bump the version, then. Made #147 to track this.

Reply:
> Oh, about the `imgplus.getColorTableCount()` truthiness: I think it would be clearer to put `> 0`. After all, it could theoretically return -1 like some C/Java code does. (OTOH, it could theoretically also return None in which case the `> 0` would raise an exception... but since the Java type signature returns `int` it can never actually return None in practice barring some kind of JPype bug shrug).

Yeah, I like this syntax better anyways

Reply:
Fixed with 11e9729